shear/cl_to_xi_fullsky/cl_to_xi_interface.py

```diff
@@ -238,6 +238,14 @@ def setup(options):
         if output_section == "":
             output_section = cl_section.replace("cl", "xi")
 
+    # Useful for assigning different names to same type of correlation function
+    # For instance, if we have measurements with two different samples
+    if save_name != "":
+        add = "_%s"%save_name
+        try:
```

Comment:
Probably want to check `if isinstance(output_section, str)` here instead of a try/except.